### PR TITLE
Backport 31575 to v7.5.x

### PR DESCRIPTION
**Pull request by bergquist**
https://github.com/grafana/grafana/pull/31575

### CODE REVIEW

File: pkg/setting/setting.go [hunk 1]
```diff
@@ -205,6 +205,9 @@ type Cfg struct {
 	RouterLogging    bool
 	Domain           string
 	CDNRootURL       *url.URL
+	ReadTimeout      time.Duration
+	EnableGzip       bool
+	EnforceDomain    bool
```
Comment (aknuds1) [inline pkg/setting/setting.go:210]: Are these added by accident? I guess they have no effect, but might be better to remove them from the PR.

Comment (bergquist) [reply pkg/setting/setting.go:210]: ah yea!. They showed up as belong to the same commit after solving the merge conflict in #31575

Comment (bergquist) [reply pkg/setting/setting.go:210]: Fine to keep em? as the cherry-pick makes more sense that way.

Comment (aknuds1) [reply pkg/setting/setting.go:210]: OK, I guess it's not important.

**Review by aknuds1 — APPROVED** [review]

**Pull request merged**